WordPress/Classes/Services/OnePasswordFacade.m

```diff
@@ -0,0 +1,27 @@
+#import "OnePasswordFacade.h"
+#import <1PasswordExtension/OnePasswordExtension.h>
+
+@implementation OnePasswordFacade
+
+- (void)findLoginForURLString:(NSString *)loginUrl viewController:(UIViewController *)viewController completion:(OnePasswordFacadeCallback)completion;
+{
+    NSParameterAssert(viewController != nil);
```

Comment:
You should had method documentation and state that this parameters cannot be nil. That will make live easier for people using this facade.